code/game/objects/structures/fitness.dm

```diff
@@ -63,3 +63,83 @@
 		else
 			to_chat(user, "<span class='notice'>Against your previous judgement, perhaps working out is not for you.</span>")
 			being_used = 0
+
+/obj/structure/fitness/boxing_ropes
```

Comment:
This needs to be moved to a _vr file, or pushed up to Polaris

Reply:
I can do that. But how do I change the map merger? I should have had the most recent version as of this afternoon.